app/app/components/AddBookToBookshelf/SearchForm.jsx

```diff
@@ -65,6 +65,23 @@ const SearchForm = ({ onAddBook }) => {
     setFilters((prevFilters) => ({ ...prevFilters, [filterType]: value }));
   };
 
+  const handleAddBook = async (book) => {
+    const isInAnyCategory = Object.values(bookShelf).some((categoryBooks) =>
+      categoryBooks.some((b) => b.google_books_id === book.google_book_id)
```

Comment:
why was the other comparison on isbn and this one is google_books?

Reply:
different editions of the same book have different google book ids so i had to go with isbn to avoid duplicates

Reply:
ahh I see :) good catch!